src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.h

```diff
@@ -6,9 +6,22 @@
 
 struct EnHorseZelda;
 
+typedef void (*EnHorseZeldaActionFunc)(struct EnHorseZelda*, GlobalContext*);
+
 typedef struct EnHorseZelda {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x15C];
+    /* 0x014C */ s32 actionIndex;
+    /* 0x0150 */ s32 animationIndex;
+    /* 0x0154 */ PSkinAwb skin;
+    /* 0x01E4 */ s32 pad;
+    /* 0x01E8 */ s32 pad2;
```

Comment:
are the pad variables completely unused? I would leave them as char arrays then since thats our universal sign for unused